jobs/ci_open_jenkins/build/bta.groovy

```diff
@@ -27,6 +27,9 @@ new SbtLibraryJobBuilder('play-partials').
 new SbtLibraryJobBuilder('play-url-binders').
         build(this as DslFactory)
 
+new SbtLibraryJobBuilder('logback-json-logger').
+        withoutJUnitReports().
```

Comment:
good question - I just copy/pasted from the existing play-json-logger..

there's no reason I can think of to suppress the junit reports so I've removed that modifier.